### PR TITLE
In-context checkout support, paypal api version update, GetTransactionId() implemented

### PR DESCRIPTION
**Pull request by gurelkaynak**
- Added option for IN_CONTEXT checkout. 
- README updated.
- Added transaction_id and GetTransactionId()

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -4,15 +4,8 @@ Pluggable PayPal NVP (Name Value Pair) API implementation for Django based web a
...
-
-    If you are using Python 2.5;
-    - /usr/local/lib/python2.5/site-packages/
-    - /usr/lib/python2.5/site-packages/
-
-    If you are using Python 2.6;
-    - /usr/local/lib/python2.6/dist-packages/
-    - /usr/lib/python2.6/dist-packages/
+    pip install -e git+django_paypal_driver_repository_url
+    All you need to do is copying the "paypal" directory (or make a symbolink) under one of the directories in your PYTHON_PATH.
```
Comment (leepa) [inline README]: This isn't good English I'm afraid.

**Comment by leepa** [comment]
Please squash these commits.

File: README (hunk 1 above)
Comment (gurelkaynak) [inline README]: that was already there... You can fix it if you want...

Comment (gurelkaynak) [inline README]: Actually it's been there for years I'm afraid...

Comment (leepa) [inline README]: You've made it make no sense however. As there's no copying if you've pip installed it.

**Comment by gurelkaynak** [comment]
squash done!

**Comment by gurelkaynak** [comment]
hey leepa, what's the problem?

**Pull request closed without merging**